tools/build-host-image.sh

```diff
@@ -117,10 +119,13 @@ function main
 
     pfexec zfs create -p rpool/images/$USER
 
+    IMAGE_NAME="$IMAGE_PREFIX: $(date +'%Y-%m-%d %H:%M:%S')"
```

Comment:
Would it be worth including the (short) git commit too?

Reply:
Yes, probably - out of omicron, helios or stlouis, which do you think would be most useful here?
I don't think helios tells you much (and it's presumably whatever is pinned in the omicron revision)
I'd prefer stlouis but it's a bit tricky to extract.

Reply:
I'm going to see how hard it is to include both omicron and host OS hashes in the image name. I think that will be most useful when looking at a list of sleds.